packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable react/forbid-foreign-prop-types */
```

Comment:
Solved in https://github.com/mui-org/material-ui/pull/22367/files#diff-e4403a877d80de653400d88d85e4801a